js/router.js

```diff
@@ -115,11 +115,14 @@ module.exports = Backbone.Router.extend({
     $('.js-navProfileMenu').removeClass('popMenu-opened');
   },
 
-  newView: function(view, bodyClass, addressBarText){
+  newView: function(view, bodyID, addressBarText, bodyClass){
     var loadingConfig;
```

Comment:
@morebrownies I've added a parameter here for any class we want to set on the body. Each route can set a body ID and a body class (only one of each). Any other ID or class on the body will be removed.

Any time you want to put an ID or Class on the body, do it here.